src/modules/FileLocksmith/FileLocksmithUI/FileLocksmithXAML/Views/MainPage.xaml

```diff
@@ -56,34 +59,35 @@
                 </ToolTipService.ToolTip>
             </Button>
 
-            <StackPanel
+            <Button
                 HorizontalAlignment="Right"
-                Orientation="Horizontal"
-                Spacing="8">
-                <Button
-                    Command="{Binding LoadProcessesCommand}"
-                    Content="{ui:FontIcon Glyph=&#xe72c;,
-                                          FontSize=16}"
-                    Style="{StaticResource SubtleButtonStyle}">
-                    <ToolTipService.ToolTip>
-                        <TextBlock x:Uid="Reload" />
-                    </ToolTipService.ToolTip>
-                </Button>
+                Command="{Binding LoadProcessesCommand}"
+                Content="{ui:FontIcon Glyph=&#xe72c;,
+                                      FontSize=16}"
+                Style="{StaticResource SubtleButtonStyle}">
+                <ToolTipService.ToolTip>
+                    <TextBlock x:Uid="Reload" />
+                </ToolTipService.ToolTip>
+            </Button>
+        </Grid>
+
+        <InfoBar
+            x:Uid="NotRunningAsAdminInfoBar"
+            Grid.Row="1"
+            Margin="16,0"
+            IsClosable="False"
+            IsOpen="{x:Bind ViewModel.IsElevated, Converter={StaticResource boolNegationConverter}, Mode=OneWay}"
+            IsTabStop="{x:Bind ViewModel.IsElevated, Converter={StaticResource boolNegationConverter}, Mode=OneWay}"
+            Severity="Informational">
+            <InfoBar.ActionButton>
                 <Button
-                    x:Name="RestartAsAdminBtn"
+                    x:Uid="RestartAsAdminBtn"
                     Command="{Binding RestartElevatedCommand}"
-                    Content="{ui:FontIcon Glyph=&#xe7ef;,
-                                          FontSize=16}"
-                    Style="{StaticResource SubtleButtonStyle}"
-                    Visibility="{x:Bind ViewModel.IsElevated, Converter={StaticResource boolToVisibilityConverter}}">
-                    <ToolTipService.ToolTip>
-                        <TextBlock x:Uid="RestartAsAdmin" />
-                    </ToolTipService.ToolTip>
-                </Button>
-            </StackPanel>
-        </Grid>
+                    Visibility="{x:Bind ViewModel.IsElevated, Converter={StaticResource boolToVisibilityConverter}}" />
```

Comment:
Do we need to set the visibility here if the `IsOpen` of the parent `InfoxBar` is already set on the same `IsElevated` property?

Reply:
Yo are right, it's not necessary. Forgot about this when moving code blocks.